base/abstractarraymath.jl

```diff
@@ -453,7 +453,7 @@ _reperr(s, n, N) = throw(ArgumentError("number of " * s * " repetitions " *
     for i in 1:length(outer)
         B = view(R, src_indices...)
         for j in 2:outer[i]
-            dest_indices[i] += inner_shape[i]
+            dest_indices[i] = dest_indices[i] .+ inner_shape[i]
```

Comment:
Couldn't this use in-place broadcast, i.e., `dest_indices[i] .= dest_indices[i] .+ inner_shape[i]`?

Reply:
NVM, I noticed that fussion would prevent `dest_indices[i] .+ inner_shape[i]` to give a range and would try to write an array instead.